### PR TITLE
[gitlab] Add documentation for publicity state of pipelines

### PR DESCRIPTION
**Pull request by Bl4d3s**
As requested by @paulmelnikow 
If anything should be written in other words etc feel free to tell me I will change it accordingly

### CODE REVIEW

**Comment by Bl4d3s** [comment]
I just looked at the staging: Maybe the image should be a lot smaller or completely removed?

**Comment by calebcartwright** [comment]
> I just looked at the staging: Maybe the image should be a lot smaller or completely removed?

Visuals are always helpful, though it is rather large. I think it may also be helpful to highlight that **Public pipelines** checkbox (perhaps with an arrow pointing to, or box around it?)

**Comment by Bl4d3s** [comment]
> > I just looked at the staging: Maybe the image should be a lot smaller or completely removed?
> 
> Visuals are always helpful, though it is rather large. I think it may also be helpful to highlight that **Public pipelines** checkbox (perhaps with an arrow pointing to, or box around it?)

I did revise the picture now its much smaller (I ommited the heading and not necessary parts) and I added a red circle /arrow to mark the important setting

File: services/gitlab/gitlab-pipeline-status.service.js [hunk 1]
```diff
@@ -15,6 +15,25 @@ const queryParamSchema = Joi.object({
   gitlab_url: optionalUrl,
 }).required()
 
+const documentation = `
+<p>
+  Important: If your project is publicly visible, but the badge is like this:
+  <img src="https://img.shields.io/badge/build-not&nbsp;found-red" alt="build not found"/>
+</p>
+<p>
+  Check, if your pipelines are publicly visible as well.<br />
```
Comment (calebcartwright) [inline services/gitlab/gitlab-pipeline-status.service.js]:
```suggestion
  Check if your pipelines are publicly visible as well.<br />
```

**Pull request merged**